src/app/core/pusher/pusher.service.ts

```diff
@@ -330,7 +330,25 @@ export class PusherService implements OnDestroy {
       if (this.isCurrentWorkspace(data)) {
         this.store$.dispatch(
           new DocumentsAction.Get({
-            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 100},
+            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 200},
+          })
+        );
+      }
+    });
+    this.channel.bind('Collection:reload', data => {
+      if (this.isCurrentWorkspace(data)) {
+        this.store$.dispatch(
+          new DocumentsAction.Get({
+            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 200},
+          })
```

Comment:
This action has filter inside for already loaded queries (look at its effect). I think we need to force refresh in this case.

Reply:
How do I do that?

Reply:
You can pass boolean value to action and use it in effect.